frontend/src/locales/en.json

```diff
@@ -109,20 +109,27 @@
       }
     },
     "collaborator": {
-      "collaboratorListItem": {
-        "cannotAssignAnotherRoleToLastManager": "You cannot assign another role to the last manager.",
-        "cannotRemoveLastManager": "You cannot remove the last manager.",
+      "collaboratorCreate": {
+        "invite": "Send invitation",
+        "inviteCta": "Invite to team",
+        "title": "Invite people"
+      },
+      "collaboratorEdit": {
+        "cannotAssignAnotherRoleToLastManager": "You cannot assign another role to the last person with administrative rights.",
+        "cannotRemoveLastManager": "You cannot remove the last person with administrative rights.",
         "deactivate": "Deactivate",
-        "resendEmail": "Resend email"
+        "delete": "Do you really want to remove collaborator '{name}'?",
+        "inviteAgain": "Invite again",
+        "resendEmail": "Resend invite",
+        "resentEmail": "Invitation sent",
```

Comment:
i would have called this key resendEmailFinished, that it differs more from resendEmail